src/html-imports.js

```diff
@@ -711,22 +718,6 @@
     return owner;
   }
 
-  // In case customElements polyfill is in use, install whenReady as the
-  // flush callback, so that custom element upgrades are deferred; this composes
-  // with any previous polyfillFlushCallback that the user may have installed
-  let prevWhenReady;
-  if (window.customElements) {
-    prevWhenReady = window.customElements['polyfillFlushCallback'];
-  } else {
-    window.customElements = {};
-  }
-  if (prevWhenReady) {
-    window.customElements['polyfillFlushCallback'] = cb => 
```

Comment:
👍  This function does not exist anymore.

BTW, when looking through the minified version of the code, it seems like the name `customElements` is being minified. I know this particular chunk is going away but it might be worth making sure other new globals are in your externs.